tests/gnrc_ipv6_nib/Makefile.ci

```diff
@@ -11,6 +11,7 @@ BOARD_INSUFFICIENT_MEMORY := \
     nucleo-l031k6 \
     stm32f030f4-demo \
     telosb \
+    msb-430 \
```

Comment:
Doesn't relate to this PR?

Reply:
Yes, seems the PR increased the binary size of the `random` library a bit which caused it to exceed the available memory on this board.